.github/workflows/pr-open.yml

```diff
@@ -16,7 +16,55 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      - name: Comment and label
+      - name: Comment, label and move to project board
         run: |
           gh pr comment ${{github.event.pull_request.number}} --body "Thank you for your contribution, our team will be reviewing this shortly, please be available for any follow up questions or code review feedback!"
           gh pr edit ${{github.event.pull_request.number}} --add-label "triage-needed"
+
+  check_files:
+    name: Check files
+    outputs:
+      run_job: ${{ steps.check_files.outputs.move_to_project }}
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout code
```

Comment:
Unfortunately, this checkout leaves us open to the type of attacks that can steal credentials. I get wanting to filter out PRs that don't change quickstarts, but I think the little bit of toil needed to remove extra PRs from the project is better than the alternative. @jpvajda Would you be fine with all PRs routing to that project?

Reply:
i agree with the approach of just including everything. however, if we really wanted to filter out files, i dont think we need to check anything out. 

similar to the translation workflow, we can just hit the public git api to see what files are changed in the pr. since the repo is public, we should be able to use a read only token / trash token to get results.

Reply:
That's a nice way to do it as well! I'd be totally fine with going that direction. The only reason I suggested just routing everything to the board is that is how the docs team handles it and it seems to work fine for them. @jpvajda  Would love to get your input on this.

Reply:
ideally we just show quickstart contributions on this board, not every PR in the repo, I think by showing all  PRS, we sorta defeat the purpose of the board, as it's supposed to aid someone in being able to quickly see all the related quickstarts activity of the project and not have to filter through all the noise of every PR

Reply:
Okay cool, then @nr-kkenney's approach should get us what we need

Reply:
👍

Reply:
@nr-kkenney will that still allow us to route the project to our board, can we use a read-only token for that as well?

Reply:
good call. i totally skipped over that in my mind 🙈 . would need write permissions for that i think.

Reply:
Ye it looks like the Docs triage uses the same method to move PR's to the board, will that need to be readdressed as well? @aswanson-nr should we default to all PR's to the board for now?

Reply:
@rudouglas that workflow was designed with all this in mind. it doesn't check out code, or operate on anything in the repository.

Reply:
oh riiiight ok cool i'll just rejig this so it follows that format. My original method for this was to send new PR's to the board if they didn't have the `eng` label on them. The idea there was that when we are creating our internal PR's we could just put that label on so only outside contributor PR's route to the board. That didn't require any checking out of code either so could also be an option

Reply:
@rudouglas You could always move this to a different workflow that only triggers on `pull_request_target` with changes to the `/quickstarts` directory. Then that would allow you to ignore the API or the fork's branch.

Reply:
big 🧠 . right there ^

Reply:
oh hello 🧠 🦢 might need a lil help on this if we have time to 🏕️ 🔥